tests/functional/snowflake_view_dependency/test_snowflake_view_dependency.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from dbt.tests.util import run_dbt, check_relations_equal
```

Comment:
doesn't Black format these on two lines as opposed to multiple imports for one line

Reply:
local runs of pre-commit are not making the change may be based off line length

Reply:
Weird! I know mine has complained at _some_ point about this so I just make sure to format them in a multiline manner.